db/view/view.cc

```diff
@@ -331,6 +331,32 @@ static query::partition_slice make_partition_slice(const schema& s) {
             std::move(opts));
 }
 
+class data_query_result_builder {
+public:
+    using result_type = query::result;
+    static constexpr emit_only_live_rows only_live = emit_only_live_rows::yes;
```

Comment:
Do you actually use this variable? I don't see where.

Reply:
I didn't dive into details, but I assumed that both this static and the result_type typedef above might be needed by the visitor pattern for some reason - e.g. I imagine a situation where the visitor type is a template parameter for the visiting function, and this function requires that these two are present. I didn't verify that assumption though and simply left these two intact just in case, perhaps they can be removed/inlined.